package.json

```diff
@@ -5,8 +5,8 @@
   "config": {
     "webservice_version": "1.15.0",
     "use_circle": true,
-    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/10135/workflows/670e8975-9acf-4211-a104-2c37b0142452/jobs/35244/artifacts",
-    "circle_build_id": "35244"
+    "circle_ci_source": "https://app.circleci.com/pipelines/github/dockstore/dockstore/10184/workflows/b5f754bc-05ff-45eb-aeed-4f71e541e087/jobs/35545/artifacts",
+    "circle_build_id": "35545"
   },
```

Comment:
These artifacts aren't from the develop branch so I'll update `package.json` after https://github.com/dockstore/dockstore/pull/5597 is merged.